TokenExtractor/AuthorizationHeaderTokenExtractor.php

```diff
@@ -17,11 +17,18 @@ class AuthorizationHeaderTokenExtractor implements TokenExtractorInterface
     protected $prefix;
 
     /**
+     * @var string
+     */
+    protected $name;
+
+    /**
      * @param string $prefix
+     * @param string $name
      */
-    public function __construct($prefix)
+    public function __construct($prefix, $name)
```

Comment:
Again, I think there should be a default value for the $name parameter, even if it is overwritable by the dependency injector like any other argument. The class is called AuthorizationHeaderTokenExtractor, so it would make sense to have it "preconfigured" for this header name. 

Maybe a constant DEFAULT_HEADER_NAME = 'Authorization' and a constructor like __construct($prefix, $name = self::DEFAULT_HEADER_NAME).

In a future major version, the class could be renamed to HeaderTokenExtractor.